pkg/cli/klothomain.go

```diff
@@ -254,7 +254,7 @@ func (km KlothoMain) run(cmd *cobra.Command, args []string) (err error) {
 			cmd.SilenceUsage = true
 		},
 	}
-	// defer analyticsClient.PanicHandler(&err, errHandler)
+	defer analyticsClient.PanicHandler(&err, errHandler)
```

Comment:
😬 lol

Reply:
yeah i saw that as i was debugging... not sure when that got commented out